firmware/port.c

```diff
@@ -84,6 +84,8 @@ void port_init(PortData* p, u8 chan, const TesselPort* port,
 
     sercom_clock_enable(p->port->spi, p->clock_channel, 1);
     sercom_clock_enable(p->port->uart_i2c, p->clock_channel, 1);
+
+    bridge_enable_chan(chan);
```

Comment:
This shouldn't turn on the power rail...`bridge_enable_chan` only sets a bit that gets sent to the host, telling it to accept connections on the channel.